Backpack/src/main/java/net/skyscanner/backpack/calendar2/data/CalendarStateMachine.kt

```diff
@@ -75,16 +84,16 @@ internal fun CalendarState.dispatchClick(date: CalendarCell.Day): CalendarState
   if (date.inactive) return this
 
   val selection = when (params.selectionMode) {
-    CalendarParams.SelectionMode.Disabled -> selection
-    CalendarParams.SelectionMode.Single -> CalendarSelection.Single(date.date)
-    CalendarParams.SelectionMode.Range -> {
+    SelectionMode.Disabled -> selection
```

Comment:
is? I tend to use is in sealed whens because the underlying entities might be converted from objects to classes in the future.

Reply:
These changes existed before my changes. But, I will fix them now.